rero_ils/modules/patrons/jsonschemas/patrons/patron-v0.0.1.json

```diff
@@ -334,6 +336,22 @@
           }
         }
       }
+    },
+    "blocked": {
+      "title": "Blocking",
+      "description": "A patron with a blocked account cannot request and borrow items, but can still renew and check in items.",
+      "type": "boolean"
+    },
+    "blocked_note": {
+      "title": "Reason",
+      "type": "string",
+      "description": "The reason is displayed in the circulation module and is visible by the patron in his account.",
+      "form": {
+        "hideExpression": "field.model.blocked !== true",
+        "expressionProperties": {
+          "templateOptions.required": "model.blocked === true"
+        }
+      }
```

Comment:
Could be interesting to have a blocked_date to know when the user has been blocked ?

Reply:
It would be probably done in a later US (as I understand)